usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js

```diff
@@ -0,0 +1,832 @@
+//
+//  markerTipEntityScript.js
+//
+//  Created by Eric Levin on 2/17/15.
+//  Additions by James B. Pollack @imgntn 6/9/2016
+//  Modifications by Thijs Wenker @thoys 1/19/2017
+//  Modified by Daniela Fontes (Mimicry) 2/9/2018
+// Modified by Rebecca Stankus 3/28/2019
+//  Copyright 2018 High Fidelity, Inc.
+//
+//  This script provides the logic for an object to draw marker strokes on its associated whiteboard
+
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+(function() {
+    var _this;
+
+    var EQUIP_MINIMUM_TIME = 1000;
```

Comment:
it seems like many of these variables are never used - do you plan to remove the unused variables?

Reply:
yes, I'm just not deleting until I've finished combining the code